### ISSUE TITLE
HTML inside Markdown table is not being render properly

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/website-archive#808]
https://software-carpentry.org/blog/2017/09/merger.html has a table that is not be translated to HTML properly.

![screenshot from 2017-09-04 17-06-00](https://user-images.githubusercontent.com/1506457/30033507-92ff4d9e-9193-11e7-8444-ca72ec54f783.png)

The table source code is at https://github.com/swcarpentry/website/pull/806/files#diff-9e722ed34f2b73ae92231f99a6eef15dR52.

# Possible solution

Use HTML table instead of Markdown table, for example

~~~
<table>
<tr>
<td>2017 Software Carpentry Steering Committee (all elected)</td>
<td>2017 Data Carpentry Steering Committee (all appointed)</td>
<td>2018 Carpentries Board of Directors (5 appointed, 4 elected)</td>
</tr>
<tr>
<td><ul><li>Rayna Harris<sup>M</sup></li><li>Kate Hertweck<sup>M</sup></li><li>Christina Koch</li><li>Mateusz Kuzak</li><li>Karin Lagesen</li><li>Sue McClatchy</li></ul><td>
<td><ul><li>Karen Cranston<sup>M</sup></li><li>Hilmar Lapp<sup>M</sup></li><li>Aleksandra Pawlik</li><li>Karthik Ram</li><li>Ethan White</li></ul></td>
<td><ul><li>Karen Cranston</li><li>Kate Hertweck</li><li>Mateusz Kuzak</li><li>Sue McClatchy</li><li>Ethan White</li><li>*to be elected*</li><li>*to be elected*</li><li>*to be elected*</li><li>*to be elected*</li></ul></td>
</tr>
</table>
~~~

# FAQ

- Is HTML inside Markdown valid? **Yes**, this could be a bug in the parser.

**Comment by tracykteal** [comment]
Thanks, I just put in a PR on the datacarpentry site with the full HTML table approach. Just waiting to see if it renders OK. Then will put in that PR here.

**Comment by tracykteal** [comment]
PR https://github.com/swcarpentry/website/pull/809

**Comment by tracykteal** [comment]
Thanks @rgaiacs. Resolved by #809
